src/test/java/org/sonarlint/intellij/config/global/SonarLintGlobalSettingsTest.java

```diff
@@ -47,10 +55,73 @@ public void testRoundTrip() {
 
   @Test
   public void testGetInstance() {
-    SonarLintGlobalSettings instance = new SonarLintGlobalSettings();
-    super.register(app, SonarLintGlobalSettings.class, instance);
-    assertThat(SonarLintGlobalSettings.getInstance()).isEqualTo(instance);
+    SonarLintGlobalSettings settings = new SonarLintGlobalSettings();
+    super.register(app, SonarLintGlobalSettings.class, settings);
+    assertThat(SonarLintGlobalSettings.getInstance()).isEqualTo(settings);
+
+  }
+
+  @Test
+  public void testLoadState() throws Exception {
```

Comment:
Nitpick: this test only checks that the old format is correctly handled, I think it would be great to add another one to check that the new format is correctly loaded/saved.